lib/checks/label/alt-empty-space.js

```diff
@@ -0,0 +1,5 @@
+let nn = node.nodeName.toLowerCase();
+let validSetup =
+	node.hasAttribute('alt') && (nn === 'img' || nn === 'input' || nn === 'area');
```

Comment:
The rule selector will not include `<input>` and `<area>` elements. Why not make this a new, separate rule altogether?

Reply:
This check doesn't need to include `input` or `area`, so that part can be removed. This only applies to image alt, so in my opinion it should not be a separate rule. `input` and `area` both use the existing check `non-empty-alt`.

Reply:
I agree with Marcy on this one. I think this one is better as a check that's part of image-alt, provided that we split off the `[role=img]`, which we have a separate support ticket for.